plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/options/TopdownNodeTypes.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.core.options;
+
+/**
+ * For general purpose topdown layout these node types specify how a node should be handled.
+ * These properties only have an effect when {@link CoreOptions#TOPDOWN_LAYOUT} is set to true.
+ * 
+ * @author mka
+ *
+ */
+public enum TopdownNodeTypes {
+    
+    /**
+     * Default type.
+     */
+    NONE,
```

Comment:
Can null or no property be used instead?

Reply:
I think the NONE is currently not actually used for anything, so that could be changed. I'm not sure what the typical approach in elk for these things would be